tests/ast/__init__.py

```diff
@@ -148,11 +148,11 @@ def test_ast_reduces_arithmetic(self):
 		self.assertEqual(statement.evaluate(None), 3)
 
 		statement = parser_.parse('one + 2', self.context)
-		self.assertIsInstance(statement.expression, ast.ArithmeticExpression)
+		self.assertIsInstance(statement.expression, ast.AddExpression)
```

Comment:
This should be in a new `test_ast_reduces_add` now instead of being in the group with arithmetic expressions.
There should also be one that test strings are reduced too.